### PR TITLE
Process mining

### PR DESCRIPTION
**Pull request by sethdaily**
@UlyssesMaclaren @camillars

- Affected routes: /consulting/process-mining

- [x] If adding a new page, I have followed the [SEO checklist](https://www.ssw.com.au/rules/seo-checklist/)

- [x] Include done video or screenshots

image
Figure: New page (content approved by Uly)

### CODE REVIEW

File: content/consulting/process-mining.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+seo:
+  title: Enterprise Development and Consulting for Process Mining
+  description: >-
+    Streamline your company processes with Power Automate Process Mining.
+    Improve your workflows, gain insights for decision making, and save costs.
```
Comment (camillars) [inline content/consulting/process-mining.mdx:6]: >     Improve your workflows, gain insights for decision making, and save costs.
1. remove 2nd comma

File: content/consulting/process-mining.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    Want to discover <span class="text-sswRed">automation opportunities</span>
+    with Process Mining?
+  subTitle: Uncover what's holding your business back with Power Automate Process Mining
+solution:
+  project: Process Mining
+callToAction: 'Talk to us about {{TITLE}} today'
+benefits:
+  benefitList:
+    - image: /images/benefits/happier-employees.png
+      title: ' AUTOMATION FOR YOUR BUSINESS'
```
Comment (camillars) [inline content/consulting/process-mining.mdx]: 2. remove extra unnecessary space before 'AUTOMATION'

File: content/consulting/process-mining.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+technologies:
+  header: Related Technologies
+  technologyCards:
+    - technologyCard: content/technologies/power-automate.mdx
+    - technologyCard: content/technologies/power-platform.mdx
+    - technologyCard: content/technologies/ai.mdx
+    - technologyCard: content/technologies/Semantic-Kernel.mdx
+---
+
+## Why **Power Automate Process Mining** is essential for your business
```
Comment (camillars) [inline content/consulting/process-mining.mdx]: 3. Fix capitalization - Follow https://www.ssw.com.au/rules/how-to-capitalize-titles

File: content/consulting/process-mining.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+<VideoEmbed url="https://youtu.be/0TcGVvAiE84" />
+
+Power Automate Process Mining is a technology that helps businesses understand their real processes, how they’re operated, and identify opportunities for improvement, automation, and digitization. It works with the existing Microsoft platform and ecosystem to provide end-to-end solutions. By extracting data from your system of records, it visualizes the processes happening in your organization. This allows you to compare processes, determine the root cause of inefficiencies, and monitor KPIs.
+
+Having mapped your processes and understood your repetitive tasks, Power Automate can help you automate the ones that cost the most time and effort.
```
Comment (camillars) [inline content/consulting/process-mining.mdx:56]: 4. Power Automate mention - add a link to the consulting page

File: content/technologies/ai.mdx [hunk 5]
```diff
@@ -7,4 +7,4 @@ readMoreSlug: 'https://www.ssw.com.au/consulting/artificial-intelligence'
 
 ### Bots/AI
```
Comment (camillars) [inline content/technologies/ai.mdx:8]: 5. Bots/AI - any internal links we should add here? https://www.ssw.com.au/consulting/artificial-intelligence

**Review by camillars — CHANGES_REQUESTED** [review]

File: content/consulting/process-mining.mdx (hunk 2 above)
Comment (sethdaily) [reply content/consulting/process-mining.mdx]: Done

File: content/consulting/process-mining.mdx (hunk 1 above)
Comment (sethdaily) [reply content/consulting/process-mining.mdx:6]: Don't think so
https://www.ssw.com.au/rules/oxford-comma/

File: content/consulting/process-mining.mdx (hunk 3 above)
Comment (sethdaily) [reply content/consulting/process-mining.mdx]: done

File: content/consulting/process-mining.mdx (hunk 4 above)
Comment (sethdaily) [reply content/consulting/process-mining.mdx:56]: as per our conversation - we don't have one

File: content/technologies/ai.mdx (hunk 5 above)
Comment (sethdaily) [reply content/technologies/ai.mdx:8]: done

**Review by camillars — APPROVED** [review]

**Pull request merged**